tools/twix/src/panels/image/mod.rs

```diff
@@ -151,36 +153,49 @@ impl Widget for &mut ImagePanel {
             self.overlays
                 .combo_box(ui, self.cycler_selector.selected_cycler());
         });
-
-        match self.show_image(ui) {
-            Ok(response) => response,
-            Err(error) => ui.label(format!("{error:#?}")),
-        }
+        let (rect, response) = ui.allocate_at_least(vec2(640.0, 480.0), Sense::click_and_drag());
+        let mut painter = TwixPainter::paint_at(ui, rect).with_camera(
+            vector![640.0, 480.0],
+            Similarity2::identity(),
+            CoordinateSystem::LeftHand,
+        );
```

Comment:
While the image panel can now use zoom and pan operations, the resizing of the image when resizing the twix panel somehow broke.

Was there a particular reason, you allocated the `TwixPainter` the way that you did, and not like for example how it is done in the map panel (from which I have taken the suggestion)? The suggestion below fixed he resizing problem and still allows for zooming and panning. 
```suggestion
        let (response, painter) = TwixPainter::allocate_new(ui);
        let mut painter = painter.with_camera(
            vector![640.0, 480.0],
            Similarity2::identity(),
            CoordinateSystem::LeftHand,
        );
```

Reply:
done